### PR TITLE
Add the top option to retrieve a limited number of records

### PR DESCRIPTION
**Pull request by d-ruiz**
Hi @hengsin, this adds the query option to request the number of items in the queried collection to be included in the result. It uses the top keyword as defined by odata.

### CODE REVIEW

File: com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/v1/resource/impl/ModelResourceImpl.java [hunk 1]
```diff
@@ -209,13 +209,16 @@ public Response getPOs(String tableName, String filter, String order, String sel
 		query.setQueryTimeout(DEFAULT_QUERY_TIMEOUT);
 		int rowCount = query.count();
 		int pageCount = 1;
-		if (rowCount > DEFAULT_PAGE_SIZE) {
-			pageCount = (int)Math.ceil(rowCount / (double)DEFAULT_PAGE_SIZE);
+		if (top > DEFAULT_PAGE_SIZE || top <= 0)
```
Comment (hengsin) [inline com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/v1/resource/impl/ModelResourceImpl.java:212]: for the > part, I think it make more sense to have a max page size instead

File: com.trekglobal.idempiere.rest.api/postman/trekglobal-idempiere-rest.postman_collection.json [hunk 2]
```diff
@@ -496,6 +496,51 @@
 			},
 			"response": []
 		},
+		{
+			"name": "api/v1/models/{table name}?top={numer of records}",
```
Comment (hengsin) [inline com.trekglobal.idempiere.rest.api/postman/trekglobal-idempiere-rest.postman_collection.json:500]: top means "number of records" ? somehow, this standard is using all naming that I don't like...

File: com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/v1/resource/impl/ModelResourceImpl.java (hunk 1 above)
Comment (d-ruiz) [reply com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/v1/resource/impl/ModelResourceImpl.java:212]: I thought about changing DEFAULT_PAGE_SIZE to ->

	private static final int MAX_PAGE_SIZE = MSysConfig.getIntValue("REST_MAX_PAGE_SIZE", 100);

Does it make sense to you? Should I add it to this pull request?

File: com.trekglobal.idempiere.rest.api/postman/trekglobal-idempiere-rest.postman_collection.json (hunk 2 above)
Comment (d-ruiz) [reply com.trekglobal.idempiere.rest.api/postman/trekglobal-idempiere-rest.postman_collection.json:500]: Yeah top means the "top" x records, is the same as limit in Postgres, I tested it with order and it is a nice addition.

**Pull request closed without merging**